Libraries/LibWeb/HTML/HTMLHeadingElement.h

```diff
@@ -26,8 +26,12 @@ class HTMLHeadingElement final : public HTMLElement {
 
     virtual Optional<String> aria_level() const override
     {
-        // TODO: aria-level = the number in the element's tag name
-        return get_attribute("aria-level"_fly_string);
+        if (auto const attr = get_attribute("aria-level"_fly_string); attr.has_value())
+            return attr;
+
+        // Implicit defaults to the number in the element's tag name.
+        auto const digit_view = local_name().bytes_as_string_view().substring_view(1, 1);
+        return String::from_utf8_without_validation(digit_view.bytes());
```

Comment:
You don't need to limit the substring to being 1 byte long - just skipping the first byte is enough, and would make it a bit more future proof in case HTML decides to add further h levels.

Also, you can write this a bit more directly. Something like:
```c++
MUST(local_name().to_string().substring_from_byte_offset(1));
```
(Working with byte counts is a bit sketchy in general, but should be fine here as we know the first character is `h` so it's a single byte.)

(You might be able to use `substring_from_byte_offset_with_shared_superstring()` but I'm not 100% confident on the FlyString->String conversion still pointing to the same data.)